pkg/operator/operator.go

```diff
@@ -88,6 +105,16 @@ func (o *Operator) Run() error {
 		<-time.After(initRetryDelay)
 	}
 
+	// Run volume provisioner
+	// The controller needs to know what the server version is because out-of-tree
```

Comment:
what if we defined a min version in a const, then actually performed a version check? that could be a separate ticket though